com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-unlit-color.md

```diff
@@ -0,0 +1,111 @@
+# URP unlit shader with color input
+
+The shader in this example adds the __Base Color__ property to the Material. You can select the color using that property and the shader fills the mesh shape with the color.
+
+Use the ShaderLab code from section [URP unlit basic shader](writing-shaders-urp-basic-unlit-structure.md) and make the following changes to it:
+
+1. Add the `_BaseColor` property definition to the Properties block:
+    
+    ```c++
+    Properties
+    { 
+        _BaseColor("Base Color", Color) = (1, 1, 1, 1)
+    }
+    ```
+    
+    This declaration adds the `_BaseColor` property with the label __Base Color__ to the Material:
+
+    ![Base Color property on a Material](Images/shader-examples/urp-material-prop-base-color.png) 
+
+    The `_BaseColor` property name is a reserved name. When you declare a property with this name, Unity uses this property as the [main color](https://docs.unity3d.com/ScriptReference/Material-color.html) of the Material. 
+
+2. After declaring a property in the Properties block, it's necessary to declare it in the HLSL program block. 
```

Comment:
When you declare a property in the Properties block, you also need to declare it in the HLSL program block.